flow/connectors/core.go

```diff
@@ -228,27 +234,27 @@ func GetConnectorAs[T Connector](ctx context.Context, config *protos.Peer) (T, e
 }
 
 func GetCDCPullConnector(ctx context.Context, config *protos.Peer) (CDCPullConnector, error) {
```

Comment:
should we remove these helpers and invoke `GetAs` directly? since we have a lot more interfaces now

Reply:
Eventually yes